invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/bulk_actions/SearchResultsRowCheckbox.js

```diff
@@ -0,0 +1,63 @@
+import { BulkActionsContext } from "./context";
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Checkbox } from "semantic-ui-react";
+import _hasIn from "lodash/hasIn";
+
+export class SearchResultsRowCheckbox extends Component {
+  static contextType = BulkActionsContext;
+
+  constructor(props) {
+    super(props);
+    this.state = { isChecked: false };
+  }
+
+  componentDidMount() {
+    this.subscribeToContext();
+    const { bulkActionContext, allSelected } = this.context;
+    this.setState({
+      isChecked: this.isChecked(bulkActionContext, allSelected),
+    });
+  }
+
+  isChecked = (bulkActionContext, allSelected) => {
+    const { rowId } = this.props;
+    if (_hasIn(bulkActionContext, `${rowId}`) || allSelected) {
+      return bulkActionContext[rowId];
+    }
+    return false;
+  };
+
+  subscribeToContext = () => {
+    const { rowId } = this.props;
+    const { allSelected, bulkActionContext } = this.context;
+    if (!_hasIn(bulkActionContext, `${rowId}`)) {
```

Comment:
Why the usage of `hasIn`?
```suggestion
    const has = bulkActionContext[rowId] !== undefined;
    if (has) {
```